releasenotes/notes/add-component-tag-299386887267885f.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
```

Comment:
This is not necessarily a change that will impact how users use the trace data, at least not yet. For someone reviewing these release notes, it's not clear what are actionable steps for them to use (or not use) the feature that's been introduced.

Reply:
I can remove the changelog if this is the case and not user impacting